src/ansys/turbogrid/core/__init__.py

```diff
@@ -23,6 +23,29 @@
 
 """PyTurboGrid is a Python wrapper for Ansys TurboGrid."""
 
+import os
+
+# When this is defined, hardcode a tg container spinup for the purpose of auto documentation.
+# Intended only for github CI runs.
+socket_port = os.getenv("PYTURBOGRID_DOC_ENGINE_CONNECTION")
+tg_container = None
+if socket_port is not None:
```

Comment:
@Acohen-work This could perhaps be hidden into its own function, to avoid the details cluttering up the __init__.py file.